integration-tests/bot/chainbot.py

```diff
@@ -348,7 +348,8 @@ async def gen_genesis(cfg):
                 "time_iota_ms": "1000"
             },
             "evidence": {
-                "max_age": "100000"
+               "max_age_num_blocks": "100000",
+               "max_age_duration": "172800000000000"
```

Comment:
What's the unit of `max_age_duration`, I think we have an assumption that `unbonding_period > max_age_duration`.

Reply:
I use the default value of the new tendermint's config. we can find it in change log:
https://github.com/tendermint/tendermint/blob/master/CHANGELOG.md